docs/guides/js-waku/light-send-receive.md

```diff
@@ -24,20 +24,25 @@ await node.start();
 When the `defaultBootstrap` parameter is set to `true`, your node will be bootstrapped using the [default bootstrap method](/guides/js-waku/configure-discovery#default-bootstrap-method). Have a look at the [Bootstrap Nodes and Discover Peers](/guides/js-waku/configure-discovery) guide to learn more methods to bootstrap nodes.
 :::
 
-A node needs to know how to route messages. In order to do that you can use standard pubsub topic (`/waku/2/default-waku/proto`). If your project uses a different pubsub topic, you can configure this by providing a set of [content topics](/learn/concepts/content-topics) for the node to use with the `contentTopics` parameter or by using the `ShardInfo` parameter:
+A node needs to know how to route messages. By default, it will use The Waku Network configuration (`{ clusterId: 1, shards: [0,1,2,3,4,5,6,7] }`). If your project uses a different network configuration, you can configure this using the `networkConfig` parameter:
 
 ```js
-// Create node with content topics
+// Create node with static sharding
 const node = await createLightNode({
   defaultBootstrap: true,
-  contentTopics: [/*set of content topics*/],
+  networkConfig: {
+    clusterId: 1,
+    shards: [0, 1, 2, 3],
+  },
 });
 
-// Create node with custom shard info
-const shardInfo = { clusterId: 3, shards: [1, 2] };
+// Create node with auto sharding
 const node = await createLightNode({
   defaultBootstrap: true,
-  shardInfo: shardInfo,
+  networkConfig: {
+    clusterId: 1,
+    contentTopics: ["/my-app/1/notifications/proto"],
```

Comment:
Shall we make it clearer that this should be the preferred method? Maybe put it first, and put static sharding config in in its won section with a title "alternative network config"